exercises/armstrong-numbers/.meta/generator/armstrong_numbers_case.rb

```diff
@@ -0,0 +1,36 @@
+require 'generator/exercise_case'
+
+class ArmstrongNumbersCase < Generator::ExerciseCase
+
+  def workload
+    case expected
+    when true  then true_workload
+    when false then false_workload
+    else raise "unexpected 'expected' value encountered"
+    end
+  end
+
+  def true_workload
+    "assert #{call_armstrong}"
+  end
+
+  def false_workload
+    "refute #{call_armstrong}"
+  end
+
+  def call_armstrong
+    delimited_input = delimit_integers(input['number'])
+    "ArmstrongNumbers.include?(#{delimited_input})"
+  end
+
+  private
+
+  def delimit_integers(integer)
+    integer
+      .digits
+      .each_slice(3)
+      .map(&:join)
+      .join('_')
+      .reverse
```

Comment:
Do we do this often enough that this should be in our available tools for generators?

Reply:
Are you asking if this should be added to the generator class, or are you saying it already exists?

Reply:
I believe @kotp was asking the former, but it seems it already exists, yes. It's called `underscore` and it's defined [here](https://github.com/exercism/ruby/blob/master/lib/generator/exercise_case/case_helpers.rb#L38-L41).

This, together with `assert_or_refute` from above should make this a 1-line `workload` method, I think.

Reply:
Okay, thank you.

Reply:
done